try:
  print(x)
except:
  print("Error is occurred due to undefined symbol x ")
finally:
  print("\nThe 'try and except' block is finished")
